src/labels/CustomLabels.labels

```diff
@@ -2107,6 +2107,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Title of the Schedule Visualizer Component</shortDescription>
         <value>Upcoming Installments</value>
     </labels>
+    <labels>
+        <fullName>RD2_StatusMustBeMapped</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message when Enhanced RD status is not mapped</shortDescription>
```

Comment:
@sam-knox Please review the new error message when RD Status is not mapped in the Status Mapping Setting

Reply:
@andrewyu-salesforce , can you please encode the quote in the custom label text?